server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAOAWSCryptoTest.java

```diff
@@ -87,8 +88,8 @@ public BlobStore testee() {
     }
 
     @Override
-    @Disabled("JAMES-2829 Not supported yet")
-    public void deleteShouldPublishDeleteTimerMetrics() {
+    @Disabled("JAMES-2829 Unstable with scality/S3 impl")
```

Comment:
Why?

Reply:
it's partial reading, if read & delete happen concurrent, sometime read operation just can read a part of the blob.

Reply:
we will find a fix and leave it disabled to have a fast merge, there are some task need this basement

Reply:
Seems sometimes we can read partially a blob that is being deleted concurrently. Couldn't find why yet but will try to deal with it in an other PR